.github/workflows/test-all-custom-flags.yml

```diff
@@ -187,8 +187,46 @@ jobs:
             ./tests-out/videos
             ./tests-out/screenshots
 
-      # Only post back to the PR if there was a failure since test-all has the
-      # typical config case covered for success feedback
+      - name: Remove past BIT status comments on originating issue
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        if: steps.set-send-pr-feedback.outputs.send_pr_feedback == 'true'
+        with:
+          github-token: ${{ steps.app-token.outputs.token }}
+          script: |
+            // Note: should match the first line of `message` in the communication steps
+            const workflowCommentTag = '<!-- comment_tag: test-all-custom-flags -->';
```

Comment:
Because these strings need to match exactly their corresponding usages in the comment steps below, we might want to utilize a shared variable pattern (for each distinct workflow) in the future.